src/test/kotlin/ls/http/LoggingTest.kt

```diff
@@ -0,0 +1,79 @@
+package ls.http
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldMatch
+import io.kotest.matchers.string.shouldStartWith
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.annotation.Client
+import io.micronaut.test.extensions.kotest5.annotation.MicronautTest
+import mu.KotlinLogging
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+@MicronautTest(transactional = false)
+class LoggingTest(@Client("/") httpClient: HttpClient) : FreeSpec({
```

Comment:
i thing it would be better to test the request id filter sperrately and just check if it is propagated to the context.

you could still have a logging test, which in general ensures that context vars are printend, however i think we do not need to necessarily test the log format. if you think, we should renam this test to sometzhing like PromtailLogging or so, and not mess with request id